Sources/Perform/UIViewController.swift

```diff
@@ -19,7 +19,7 @@ extension UIViewController {
 
       return AnyIterator {
         if let next = queue.popLast() {
-          queue.insert(contentsOf: next.childViewControllers, at: 0)
+          queue.insert(contentsOf: next.children, at: 0)
```

Comment:
Can we back-port the `children` spelling using a compile time–conditional extension?